yt_dlp/extractor/camsoda.py

```diff
@@ -0,0 +1,57 @@
+import random
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, traverse_obj
+
+
+class CamsodaIE(InfoExtractor):
+    _VALID_URL = r'https?://www\.camsoda\.com/(?P<id>[0-9A-Za-z-]+)'
+    _TESTS = [{
+        'url': 'https://www.camsoda.com/lizzhopf',
+        'info_dict': {
+            'id': 'lizzhopf',
+            'ext': 'mp4',
+            'title': 'lizzhopf (lizzhopf) Nude on Cam. Free Live Sex Chat Room - CamSoda',
+            'description': str,
+            'is_live': True,
+            'age_limit': 18,
+        },
+        'skip': 'Room is offline',
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        user_id = f'guest_{random.randrange(10000, 99999)}'
+        webpage = self._download_webpage(url, video_id, headers=self.geo_verification_headers())
```

Comment:
is it worth downloading webpage just for description?

Reply:
What about to also extract the title to address your other comment?

Reply:
I was just asking. I don't use the site - it's your discretion whether the metadata is worth the additional request

Reply:
I don't use this metadata either, but if this metadata is useful to someone else, it does look like retrieving the page is the only way to get it so I think it makes sense.